app/components/candidate_interface/new_reference_status_line_component.rb

```diff
@@ -0,0 +1,17 @@
+module CandidateInterface
+  class NewReferenceStatusLineComponent < NewReferenceHistoryComponent
+    attr_reader :reference
+
+    def request_bounced_event
+      history.find { |event| event.name == 'request_bounced' }
+    end
+
+    def request_sent_event
+      history.find { |event| event.name == 'request_sent' }
+    end
```

Comment:
I think these should just reference the `feedback_status` state of the reference, rather than search through the history?

Reply:
👌